package.json

```diff
@@ -29,7 +29,7 @@
     "genloc": "java -classpath swagger-codegen-cli.jar io.swagger.codegen.SwaggerCodegen generate -i http://127.0.0.1:16043/rest/v1/swagger.json -l typescript --additional-properties classPrefix=icc -o ./icc-api",
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "tsc",
-    "prepare": "rimraf -rf dist && npm run build",
+    "prepare": "rimraf dist && npm run build",
```

Comment:
@cebamps: ok for you?

Reply:
Sure. That seems to be the intended way to call rimraf.